src/main/java/org/apache/commons/lang3/math/NumberUtils.java

```diff
@@ -647,6 +651,9 @@ public static BigDecimal toScaledBigDecimal(final String value, final int scale,
      * @return Number created from the string (or null if the input is null)
      * @throws NumberFormatException if the value cannot be converted
      */
+    @SuppressWarnings("argument.type.incompatible")/*
+    #1, #2, #3: !StringUtils.isBlank(str) => str.length != 0
```

Comment:
`StringUtils.isBlank()` should be annotated with an `@EnsuresMinLenIf` annotation that says that if it returns false, the string is `@MinLen(1)`